svn/common.py

```diff
@@ -423,11 +424,15 @@ def diff(self, old, new, rel_path=None):
             ['--old', '{0}@{1}'.format(full_url_or_path, old),
              '--new', '{0}@{1}'.format(full_url_or_path, new)],
             combine=True)
-        file_to_diff = {i.split('==')[0].strip(): i.split('==')[-1].strip().strip('=').strip()
-                        for i in filter(None, diff_result.split('Index: '))}
+        file_to_diff = \
+            {
+                i.split('==')[0].strip(): i.split('==')[-1].strip('=').strip()
```

Comment:
See commit on previous commit. I'd prefer this be expanded into a tradition *for*.